lint

```diff
@@ -8,10 +8,14 @@
   "license": "MIT",
   "devDependencies": {
     "jest": "^23.4.1",
-    "prettier": "1.13.7"
+    "prettier": "^1.13.7",
+    "prettier-check": "^2.0.0"
   },
   "scripts": {
     "test": "jest",
+    "test2": "prettier-check src/*.js",
```

Comment:
Can you rename this to `lint` please?

I did not express that well in https://github.com/sourcelair/env-spec-js/issues/9 😅.

Reply:
the file's name will be lint or package.json?

Reply:
The file's name should be `package.json`.

Inside the `scripts` section of the `package.json` file we will need the following entry:

```json
    "lint": "prettier-check src/*.js",
```